schematic/utils/csv_utils.py

```diff
@@ -208,14 +210,11 @@ def create_schema_classes(schema_extension: pd.DataFrame, se: SchemaExplorer) ->
 
     try:
         check_schema_definition(schema_extension)
-        print("Schema definition csv ready for processing!")
+        logger.info("Schema definition csv ready for processing!")
     except:
-        print("Schema extension headers: ")
-        print(set(list(schema_extension.columns)))
-        print("do not match required schema headers: ")
-        print(required_headers)
-        print("ERROR: could not add extension to schema!")
-        exit()
+        logger.debug(f"Schema extension headers: {set(list(schema_extension.columns))} "
+                    f"do not match required schema headers: {required_headers}")
+        raise ValueError("The extension could not be added to the schema.")
```

Comment:
Use `logger.exception()` instead of `logger.debug()`. And make error message more descriptive.